common/nymcoconut/src/scheme/keygen.rs

```diff
@@ -316,7 +317,7 @@ impl VerificationKey {
         let beta_g1_len = self.beta_g1.len() as u64;
         let beta_g2_len = self.beta_g2.len() as u64;
         let mut bytes = Vec::with_capacity(
-            8 + (beta_g1_len + 1) as usize * 48 + (beta_g2_len + 1) as usize + 96,
+            96 + 8 + beta_g1_len as usize * 48 + beta_g2_len as usize * 96,
         );
```

Comment:
Why convert to `u64` and then later return to `usize`? It makes the operation harder to read and it's not really needed